portal/index.php

```diff
@@ -76,7 +76,7 @@
  * */
```

Comment:
note to self:
need to review this script outside PR to view it with white changes ignored

Reply:
oops, just needed to use ?w=1 trick, resolved